WTWSMS/common/landed_titles/WtWSMS_holy_orders.txt

```diff
@@ -286,4 +286,65 @@ d_wasilla_guardians = {
 	# Extra income due to donations, etc
 	monthly_income = 10
 
+}
+
+c_priscillianism_holy_order = {
+		color = { 0.5 0.7 0.5 }
+		
+		graphical_culture = occitangfx # Closest thing to Iberian GFX in the game
+		
+		capital = 198 # Avila
+		
+		title = "PRISCMASTER"
+		foa = "HOCHMEISTER_FOA"
+		
+		landless = yes
+		
+		holy_order = yes
+		
+		religion = priscillianism
+		
+		# culture = cantabrian
+		
+		primary = yes
+		
+		short_name = yes
+		
+		name_tier = DUKE
+		
+		monthly_income = -0.5 # Asceticism (this is supposed to reduce the income by half)
```

Comment:
Are you sure on that? Because that negative one would be applied regardless of whether or not they gain money.